src/test/java/org/apache/maven/it/VerifierTest.java

```diff
@@ -105,19 +101,20 @@ public void testStripAnsi()
     @Test
     public void testLoadPropertiesFNFE() throws VerificationException
     {
-        exception.expectCause( isA( FileNotFoundException.class ) );
-
-        Verifier verifier = new Verifier( "src/test/resources" );
-        verifier.loadProperties( "unknown.properties" );
+        VerificationException exception = assertThrows( VerificationException.class, () -> {
+            Verifier verifier = new Verifier( "src/test/resources" );
+            verifier.loadProperties( "unknown.properties" );
+        } );
+        assertInstanceOf( FileNotFoundException.class, exception.getCause() );
     }
 
     @Test
     public void testDedicatedMavenHome() throws VerificationException, IOException
     {
         String mavenHome = Paths.get( "src/test/resources/maven-home" ).toAbsolutePath().toString();
-        Verifier verifier = new Verifier( temporaryFolder.getRoot().toString(), null, false, mavenHome );
+        Verifier verifier = new Verifier( temporaryFolder.toString(), null, false, mavenHome );
```

Comment:
That's ugly, the ctor should rather accept `Path`

Reply:
There are 10 different constructors, I'm not sure it's a good idea to add 10 more that would accept a `Path` instead of a `String` as the first parameter.